src/coreclr/src/pal/src/init/pal.cpp

```diff
@@ -359,18 +359,34 @@ Initialize(
         gPID = getpid();
         gSID = getsid(gPID);
 
+        // Initialize the TLS lookaside cache
+        if (FALSE == TLSInitialize())
+        {
+            palError = ERROR_PALINIT_TLS;
+            goto done;
+        }
+
+        // Initialize debug channel settings before anything else.
+        // This depends on the environment, so it must come after
+        // EnvironInitialize.
```

Comment:
EnvironInitialize is still way below. Is this comment right; or EnvironInitialize needs to be moved up as well?

Reply:
Oops, forgotten to delete that portion of the comment. It is not needed as I've got rid of that dependency.

Reply:
Fixed